Sources/AppStoreConnectClient/Models/BuildsQuery.swift

```diff
@@ -27,7 +27,63 @@ import Foundation
 /// A structure representing a query for builds with specified sort and fields parameters.
 public struct BuildsQuery {
     /// The sort payload for the query.
-    let sort: Operations.builds_hyphen_get_collection.Input.Query.sortPayload
+    public let sort: [Sort]?
     /// The fields payload for the query.
-    let fields: Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_Payload
+    public let fields: [Fields]?
+    
+    /// Converts the Sort enum to the appropriate payload type used in the query.
+    /// - Parameter from: The Sort enum value to convert.
+    /// - Returns: The converted sort payload.
+    func convertSort(_ from: [Sort]?) -> [Operations.builds_hyphen_get_collection.Input.Query.sortPayloadPayload]? {
+        return from?.compactMap { sort in
+            return Operations.builds_hyphen_get_collection.Input.Query.sortPayloadPayload(rawValue: sort.rawValue)
+        }
+    }
+    
+    /// Converts the Fields enum array to the appropriate payload type used in the query.
+    /// - Parameter from: The array of Fields enum values to convert.
+    /// - Returns: An array of converted field payloads.
+    func convertFields(_ from: [Fields]?) -> [Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_PayloadPayload]? {
```

Comment:
`func convertFields(_ from: [Fields]?) -> [Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_PayloadPayload]?`
it is so-so decision. Need something like this `func convert(from fields: [Fields]) -> [Operations.builds_hyphen_get_collection.Input.Query.fields_lbrack_builds_rbrack_PayloadPayload]`

Reply:
Changed.